docs/compatibility.md

```diff
@@ -441,6 +441,42 @@ parse some variants of `NaN` and `Infinity` even when this option is disabled
 ([SPARK-38060](https://issues.apache.org/jira/browse/SPARK-38060)). The RAPIDS Accelerator behavior is consistent with
 Spark version 3.3.0 and later.
 
+### get_json_object
+
+The `GetJsonObject` operator that takes JSON input and a json path string to process. The code base
+for this is currently separate from regular JSON parsing and so it has other problems associated
+with it, in addition to potential problems with parsing the json path string itself. Because of this
+`GetJsonObject` has been disabled to be on the GPU by default. But if you are aware of the current
```

Comment:
Nit: `disabled to be on the GPU by default` reads a bit like it's going to be on the GPU by default.
```suggestion
`GetJsonObject` will be on the CPU by default. But if you are aware of the current
```